client/my-sites/importer/importing-pane.jsx

```diff
@@ -194,9 +195,22 @@ export default React.createClass( {
 	},
 
 	render: function() {
-		const { site: { ID: siteId, name: siteName, single_user_site: hasSingleAuthor } } = this.props;
-		const { importerId, errorData = {}, customData } = this.props.importerStatus;
+		const {
+			importerStatus: {
+				importerId,
+				errorData = {},
+				customData
+			},
+			mapAuthor: mapAuthorFor,
```

Comment:
I vote for renaming it to `mapAuthorFor` in `mapDispatchToProps` :) We are wrapping it with dispatch function call and two thunks anyway.